### PR TITLE
python script for scraping and posting

### PR DESCRIPTION
**Pull request by blopit**
sample script python2.7.6

### CODE REVIEW

File: scraper.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+__author__ = 'shrenilpatel'
+
+import json
+import urllib2
+
+#FOOD SERVICE
+getUrl = "https://api.uwaterloo.ca/v2/foodservices/locations.json?key=a19ad7d4ee67feebd6b7caa461251302"
+#postUrl = 'https://geotaganything.herokuapp.com/tags' #ACTUAL
```
Comment (rjoleary) [inline scraper.py:8]: I'm assuming posting to geotaganything.herokuapp.com won't work for this script due to authentication -- that's why it's commented out. Correct?

File: scraper.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        image = item['logo']
+        subcat = None
+
+        data = {
+            'latitude': lat,
+            'longitude': lon,
+            'name': name,
+            'description': desc,
+            'category': cat,
+            'image': image
```
Comment (rjoleary) [inline scraper.py:30]: Can you make all entries generated for uwapi have the username "uwapi", so updating and managing them all is easier. I noticed right now, it's null.

File: scraper.py (hunk 1 above)
Comment (blopit) [reply scraper.py:8]: yea currently, but then again it shouldnt work on local either

File: scraper.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+__author__ = 'shrenilpatel'
+
+import json
+import urllib2
+
+#FOOD SERVICE
+getUrl = "https://api.uwaterloo.ca/v2/foodservices/locations.json?key=a19ad7d4ee67feebd6b7caa461251302"
+#postUrl = 'https://geotaganything.herokuapp.com/tags' #ACTUAL
+postUrl = 'http://localhost:5000/tags' #LOCAL
```
Comment (rjoleary) [inline scraper.py:9]: How does posting to this URL work exactly? I've never used Heroku. Do you have to somehow ssh into the server to run this Python script?

File: scraper.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+        print data
+        ret.append(data)
+
+    return ret
+
+def post(data, url):
+    req = urllib2.Request(url)
+    req.add_header('Content-Type', 'application/json')
+    response = urllib2.urlopen(req, json.dumps(data))
```
Comment (rjoleary) [inline scraper.py:41]: We should check the response, but life's too short.

**Review by rjoleary — APPROVED** [review]
TLDR: life's too short, approved

File: scraper.py (hunk 3 above)
Comment (blopit) [reply scraper.py:9]: No this was just a quick and dirty example to get from UWAPI and post to Heroku. API has changed so this all doesnt really work unless your local is running the version before API changes. It's not very useful ATM but its a start for anyone interested in the task. The idea was you'd change the getUrl and get method and just run the script once. Though there is benefit from keeping them all
